app/forms/provider_interface/offer_wizard.rb

```diff
@@ -148,5 +181,22 @@ def update_path_history(attrs)
       @wizard_path_history.update
       @path_history = @wizard_path_history.path_history
     end
+
+    def setup_further_conditions
+      self.further_conditions = 4.times.map do |index|
+        existing_value = further_conditions && further_conditions[index]
+        existing_value.presence || ''
+      end
+    end
+
+    def further_conditions_valid
+      conditions_models.map(&:valid?).all?
```

Comment:
This, because we need to run `valid?` on all of the conditions, rather than returning at the first invalid one